src/OrchardCore.Modules/OrchardCore.Markdown/ResourceManagementOptionsConfiguration.cs

```diff
@@ -29,7 +26,7 @@ static ResourceManagementOptionsConfiguration()
 
         _manifest
             .DefineScript("easymde-mediatoolbar")
-            .SetDependencies("easymde")
+            .SetDependencies("easymde", "jQuery")
```

Comment:
Why are the `depends-on` changes in the cshtml files necessary after this?

Reply:
it is not. but I added them because the inline script itself also depend on jQuery. This way if we ever remove `jQuery` from here it won't break the inline script.